### PR TITLE
Add F-Droid

### PR DESCRIPTION
**Pull request by ghost**
Fix part of #25905 
Signed-off-by: HouraisanNEET

### CODE REVIEW

File: lib/public/Defaults.php [hunk 1]
```diff
@@ -96,6 +96,15 @@ public function getAndroidClientUrl() {
 		return $this->defaults->getAndroidClientUrl();
 	}
 
+	/**
+	 * link to the Android client on F-Droid
+	 * @return string
+	 * @since ?
```
Comment (HouraisanNEET) [inline lib/public/Defaults.php:102]: Not sure when it will be released.

Comment (gary-kim) [reply lib/public/Defaults.php:102]: Set it to `@since 23.0.0` to signify Nextcloud 23.

**Review by gary-kim — APPROVED** [review]
Maybe that svg could be compressed a bit but otherwise, seems fine to me.

**Review by juliusknorr — APPROVED** [review]

File: lib/private/legacy/OC_Defaults.php [hunk 2]
```diff
@@ -151,6 +153,18 @@ public function getAndroidClientUrl() {
 		}
 	}
 
+	/**
+	 * Returns the URL to Google Play for the Android Client
```
Comment (wonderfulShrineMaidenOfParadise) [inline lib/private/legacy/OC_Defaults.php:157]: "Returns the URL to Google Play for the Android Client"
s/Google Play/F-Droid/g

**Pull request merged**